APMrover2/AP_MotorsUGV.cpp

```diff
@@ -202,8 +208,8 @@ bool AP_MotorsUGV::have_skid_steering() const
     return false;
 }
 
-// returns true if omni rover
-bool AP_MotorsUGV::is_omni_rover() const
+// returns true if vehicle is capable of lateral movement
+bool AP_MotorsUGV::has_lateral_control() const
```

Comment:
on a more generale not, I won't restraint lateral control to omni only. I think we should keep the omni handling as it. I totally see the use of direct lateral control on boat with lateral motor pods for example.
So I pretty again this change, the omni_rover is doing complex mixing and not only adding direct lateral control !

Reply:
At the moment, "is_omni_rover" and "has_lateral_control" are the same.  I think once we add thruster support for boats we can come back and separate the two features.

Reply:
@khancyr boats with vectored motor configuration is actually the next step! My thinking was that we could have 1 method for all frames that support lateral input, of course it would have to change every time a new frame is added, but you do have a point about omni rover being different from frames with direct lateral input so perhaps it should have a separate method for now.